sql/org_mozilla_firefox_derived/incline_executive_v1/query.sql

```diff
@@ -41,23 +33,43 @@ WITH fennec_client_info AS (
   LEFT JOIN
     `moz-fx-data-shared-prod.org_mozilla_firefox.migrated_clients` migrated_clients
   ON
+    -- For Fennec, we only want to look at historical migration pings
+    -- to see if this client has migrated. We use this to check if they
+    -- were migrated today as well.
     clients_last_seen.client_id = migrated_clients.fennec_client_id
+    AND clients_last_seen.submission_date >= migrated_clients.submission_date
   WHERE
-    clients_last_seen.submission_date = @submission_date
-    AND (
-      migrated_clients.submission_date = @submission_date
-      OR migrated_clients.submission_date IS NULL)
+    clients_last_seen.submission_date IN (
+      @submission_date,
+      DATE_SUB(@submission_date, INTERVAL 1 YEAR),
+      DATE_SUB(DATE_SUB(@submission_date, INTERVAL 1 YEAR), INTERVAL 1 WEEK)
+    )
```

Comment:
This condition definitely could use some comments. Do we really mean to include (1 year + 1 week)? Or is this supposed to be today, 1 week ago, and 1 year ago?

Reply:
Can we completely avoid this logic if we backfill this table to 2019-01-01?

Reply:
We can, but I would much prefer not to. We are still iterating on this and a year+ backfill would slow us down prohibitively.